keras_cv/layers/preprocessing/random_cutout.py

```diff
@@ -194,31 +199,21 @@ def _compute_rectangle_position(self, inputs):
             input_shape[2],
         )
         center_x = tf.random.uniform(
-            shape=[batch_size],
+            shape=[batch_size, self.num_cutouts],
```

Comment:
self.num_cutouts is baked into the graph here.  You'll need to leverage `get_random_transformation` to handle the case where ranges are supported.  You can see an example of how this works in `random_shear.py`